cub/cub/util_macro.cuh

```diff
@@ -137,6 +137,9 @@ _CCCL_DIAG_SUPPRESS_CLANG("-Wattributes")
 #if !defined(_CCCL_CUDA_COMPILER_NVHPC)
 _CCCL_DIAG_SUPPRESS_NVHPC(attribute_requires_external_linkage)
 #endif // !_CCCL_CUDA_COMPILER_NVHPC
+#if defined(__ICC)
+#pragma warning(disable: 1890)
```

Comment:
Important; Please add a comment what exact warning is suppressed here, its difficult to remember all those
```suggestion
#pragma warning(disable: 1890) // the "__visibility__" attribute can only appear on functions and variables with external linkage
```

Reply:
The suppression is under `#if !defined(CUB_DISABLE_KERNEL_VISIBILITY_WARNING_SUPPRESSION)`, which seems to indicate the purpose.

Reply:
Does it hurt to add it? I really want to know what `#pragma warning` does without looking at surrounding code